scylla/tests/ccm-integration/node_config.rs

```diff
@@ -0,0 +1,365 @@
+use core::str;
+use serde_yaml::Value;
+use std::collections::HashMap;
+
+/// Represents data stored in scylla.yaml or cassandra.yaml
+#[derive(Debug, Clone)]
+pub(crate) enum NodeConfig {
+    Null,
+    Bool(bool),
+    Int(i64),
+    Float(f64),
+    String(String),
+    List(Vec<NodeConfig>),
+    Map(HashMap<String, NodeConfig>),
+}
+
```

Comment:
This is basically the same as `serde_yaml::Value`. Why did you need to do this, instead of having a newtype `NodeConfig(Value)`?